test_schedule_file.py

```diff
@@ -0,0 +1,80 @@
+"""Schedule Fixed Interval Schema"""
```

Comment:
Is this a test file?  I don't see any assertions, also odd that `Date` and other classes are being redefined here?